doc/fast-sync.md

```diff
@@ -12,13 +12,13 @@ In short, a fast-sync in Grin does the following:
 
 1. Download all block headers, by chunks, on the most worked chain, as
    advertized by other nodes.
-1. Find a header sufficiently back from the chain head. This is called the node
+2. Find a header sufficiently back from the chain head. This is called the node
    horizon as it's the furthest a node can reorganize its chain on a new fork if
    it were to occur without triggering another new full sync.
-1. Download the full state as it was at the horizon, including the unspent
+3. Download the full state as it was at the horizon, including the unspent
    output, range proof and kernel data, as well as all corresponding MMRs. This is
    just one large zip file.
-1. Validate the full state.
-1. Download full blocks since the horizon to get to the chain head.
+4. Validate the full state.
+5. Download full blocks since the horizon to get to the chain head.
```

Comment:
Using `1.` everywhere in markdown allows us to simply ignore the numbering and it will be done automatically.

Reply:
Yes, I know. But I read other docs before I made these changes.

Anyway, I change all to `1.` besides [contract_ideas.md](https://github.com/mimblewimble/grin/blob/master/doc/contract_ideas.md)(for crossing line numbering).